### ISSUE TITLE
Chinese in  code generate

### ISSUE THREAD

**Issue by zhanglzu** [THUDM/ChatGLM-6B#272]
### Is there an existing issue for this?

- [x] I have searched the existing issues

###
ChatGLM-6B：好的，下面是一个简单的 PyTorch 代码示例，用于掩码一个序列中的某些元素，并输出掩码后的结果。这个示例使用了 PyTorch 中的 Transformer 模型。

import torch
import torch.nn as nn
import torch.nn.functional as F

class掩码Layer(nn.Module):
    def __init__(self, input_size, hidden_size, output_size):
        super(掩码Layer, self).__init__()
        self.transformer = nn.TransformerLayer(input_size, hidden_size, output_size)
        self.transformer.embedding = nn.Embedding(input_size, embedding_size)
        self.transformer.layer = self.transformer
        self.transformer.pos_编码 = nn.Pos编码

    def forward(self, x):
        x = self.transformer.embedding(x)
        out, (hidden, attention) = self.transformer.layer(x)
        out = self.transformer.pos_编码(out[:, :attention.size(0)])
        return F.relu(out[:, attention.size(0)])

def掩码(input_序列， mask):
    input_序列 = input_序列.view(-1, input_序列.size(0))
    output =掩码Layer(input_序列， hidden_size, output_size).forward(input_序列)
    return output[mask], output[mask]

# 示例
input_序列 = torch.randn(100, 10)
mask = torch.randint(0, 100, size=input_序列.size(0))
output =掩码(input_序列， mask)
print(output)
###

### Steps To Reproduce

when I ask Chatglm to generate codes introduing mask to transformer model, while Chinese would appear in Codes

### Environment

```markdown
- OS:window 10
- Python: 3.9
- Transformers:4.26
- PyTorch:1.13
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by yhyu13** [comment]
lol, 我不是开发者，但是py3支持中文变量啊。一般不会出现就是了。可能需要手动提示一下

**Comment by zhanglzu** [comment]
我在后面的提问中也说明了，请把代码中的中文改为英文，但是结果还是这样。可能把问题改为：把中文变量改为英文变量，可能能成功。有需要的可以试试。

**Comment by starplatinum3** [comment]
大佬 请问你是 Python: 3.9. 几，3.9 哪个稳定呀
